### PR TITLE
Chapter 4 & 5

### PR DESCRIPTION
**Pull request by NurAnisAqilah**
ready

### CODE REVIEW

**Comment by NurAnisAqilah** [comment]
ready

File: app/views/layouts/_header.html.erb [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+      <nav>
+       <ul class="nav navbar-nav navbar-right">
+         <li><%= link_to "Home", "#" %></li>
+         <li><%= link_to "Help", "#" %></li>
+         <li><%= link_to "Log in", "#" %></li>
+       </ul>
+     </nav>
+   </div>
+ </header>
+
```
Comment (saobv-2223) [inline app/views/layouts/_header.html.erb]: image

**Review by saobv-2223 — COMMENTED** [review]
image

**Comment by NurAnisAqilah** [comment]
ready

File: app/assets/stylesheets/custom.scss [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+.center h1 {
+    margin-bottom: 10px;
+}
+
+/* header */
+#logo {
+    float: left;
+    margin-right: 10px;
+    font-size: 1.7em;
+    color: #fff;
```
Comment (chungpv-1008) [inline app/assets/stylesheets/custom.scss:27]: please create color variable in other file, then call back here

File: app/assets/stylesheets/custom.scss [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    text-transform: uppercase;
+    letter-spacing: -1px;
+    padding-top: 9px;
+    font-weight: bold;
+    }
+    #logo:hover {
+    color: #fff;
+    text-decoration: none;
+    }
```
Comment (chungpv-1008) [inline app/assets/stylesheets/custom.scss]:
```suggestion
    #logo:hover {
      color: white;
      text-decoration: none;
    }
```

**Review by chungpv-1008 — COMMENTED** [review]
Please fix comment, then merge this pull request yourself

File: app/assets/stylesheets/custom.scss (hunk 2 above)
Comment (NurAnisAqilah) [reply app/assets/stylesheets/custom.scss:27]: may I know which file should I create color variable?

**Pull request merged**